### DISCUSSION TITLE
Accessing from Iran

### DISCUSSION THREAD

**Discussion by Abolfazl-MI** [pockethost/pockethost#206] [Q&A]
hello team
I'm  interstate to use your hosting for my MVP flutter application , I need something like Firebase but unfortunately
the Firebase didn't allow Iranian user to use it so I found  your service alternative for it but I could not use it in dashboard and other place 
did your team has policy as Firebase? are we banned to use ?
this is so important for me please answer it :heart:

**Comment by benallfree** [comment]
@Abolfazl-MI We don't have any policies about regional access, but we may in the future. 

> but I could not use it in dashboard and other place

I'm not sure what you mean, can you please say more?

**Comment by Abolfazl-MI** [comment]
@benallfree  thanks for your response , I cant access the Dashboard or other part of site without VPN , I should connect to america or other country to have access to host

**Reply by benallfree** [reply]
It sounds like something might be blocked on your end. We aren't blocking anyone.

**Reply by Abolfazl-MI** [reply]
Thanks for your respond , Its sweets of you for responding . unfortunately our country in technology bans everything and we as developers facing this thing every day and we are living at the hardest and harshest situation ever occur in history :( hope for better days

**Comment by X7md** [comment]
Hi, try use this to bypass DPI

https://github.com/xvzc/SpoofDPI
